docs/API.md

```diff
@@ -548,16 +548,16 @@ $ curl -X POST --header "Content-Type:application/json" \
 
 > **Note:** on devices with supervisor version lower than 7.22.0, replace all `BALENA_` variables with `RESIN_`, e.g. `RESIN_SUPERVISOR_ADDRESS` instead of `BALENA_SUPERVISOR_ADDRESS`.
 
-This endpoint allows setting some configuration values for the host OS. Currently it supports
-proxy and hostname configuration.
+This endpoint allows setting some configuration values for the host OS. Currently it supports proxy and hostname configuration.
 
 For proxy configuration, balenaOS 2.0.7 and higher provides a transparent proxy redirector (redsocks) that makes all connections be routed to a SOCKS or HTTP proxy. This endpoint allows services to modify these proxy settings at runtime.
 
 
 #### Request body
 
-Is a JSON object with several optional fields. Proxy and hostname configuration go under a "network" key. If "proxy" or "hostname" are not present (undefined), those values will not be modified, so that a request can modify hostname
-without changing proxy settings and viceversa.
+A JSON object with several optional fields. Proxy and hostname configuration go under a `network` key. If `proxy` or `hostname` are not present (undefined), those values will not be modified, so that a request can modify hostname without changing proxy settings and vice versa.
+
+By default, with [balenaOS 2.82.6](https://github.com/balena-os/meta-balena/blob/master/CHANGELOG.md#v2826) or newer, host config PATCH requests will cause a balenaEngine restart, so the request will respect the presence of update locks. Specify the `force` boolean property in the request body to ignore this.
```

Comment:
```
By default, with [balenaOS 2.82.6](https://github.com/balena-os/meta-balena/blob/master/CHANGELOG.md#v2826) or newer, host config host config PATCH requests may cause a balena engine restart, in order for the container to load the new network configuration, so the request will respect the presence of update locks```
```

Probably this line should include the supervisor version as well

Reply:
Also 

```
Specify the `force` boolean property (`false` by default) ...
```

Reply:
> Probably this line should include the supervisor version as well

You mean the Supervisor version that this PR will be? The Supervisor version that comes with 2.82.6 will not include the update lock check.

Reply:
I've force pushed a docs change which specifies that Supervisor v12.11.34 and newer will check for update locks.